app/views/categories/show.html.erb

```diff
@@ -11,16 +11,18 @@
 <% end %>
 
 <% post_count = @posts.count %>
-<div class="has-color-tertiary-500 category-meta" title="<%= post_count %>">
-  <div>
+<div class="has-color-tertiary-500 category-meta">
+  <div title="<%= post_count %> posts">
     <%= short_number_to_human post_count, precision: 1, significant: false %>
-    <%= 'post'.pluralize(post_count) %> &middot;
-
+    <%= 'post'.pluralize(post_count) %> 
+  </div>
```

Comment:
Once the extra layer of containment has been added, this inner div around just the post count will cause it to be on a new line, rather than the same line as the Edit Category link and the Subscribe button.

This is because a div is a block element that defaults to being on its own line. If you change it to a span (which is an inline element) it will default to being on the same line as its neighbours.